src/js/netjsongraph.render.js

```
@@ -241,7 +240,7 @@ class NetJSONGraphRender {
     const linesData = [];
     let nodesData = [];
 
-    nodes.forEach((node) => {
+    nodes.forEach((node, index) => {
```

ESLint

[error] 'index' is defined but never used.

(no-unused-vars)